libs/cimit-service/src/test/java/uk/gov/di/ipv/core/library/service/CiMitServiceTest.java

```diff
@@ -449,6 +452,19 @@ void getContraIndicatorsVCJwtThrowsErrorWhenInvalidCimitKey() throws JsonProcess
                                 .statusCode(200)
                                 .payload(makeCiMitVCPayload(SIGNED_CONTRA_INDICATOR_VC))
                                 .build());
+        when(verifiableCredentialValidator.parseAndValidate(
+                        eq(TEST_USER_ID),
+                        eq(null),
+                        eq(SIGNED_CONTRA_INDICATOR_VC),
+                        eq(VerifiableCredentialConstants.SECURITY_CHECK_CREDENTIAL_TYPE),
+                        any(),
+                        eq("INVALID_CIMIT_KEY"),
+                        eq(CIMIT_COMPONENT_ID),
+                        eq(false)))
+                .thenThrow(
+                        new VerifiableCredentialException(
+                                HTTPResponse.SC_SERVER_ERROR,
+                                ErrorResponse.FAILED_TO_VALIDATE_VERIFIABLE_CREDENTIAL));
```

Comment:
It seems odd that you had to add this, but perhaps it should always have been here?
Not something you caused, but could these tests work with a real verifiableCredentialValidator instead of setting up a mock each time?